File: Dashboard Template/vyzor-symfony/final/src/Controller/MapsController.php
<?php

namespace App\Controller;

use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
use Symfony\Component\HttpFoundation\Response;
use Symfony\Component\Routing\Annotation\Route;

class MapsController extends AbstractController
{

    #[Route('/google-maps', name: 'app_google_maps')]
    public function googleMaps(): Response
    {
        return $this->render('maps/google-maps.html.twig', [
            'controller_name' => 'MapsController',
        ]);
    }

    #[Route('/leaflet-maps', name: 'app_leaflet_maps')]
    public function leafletMaps(): Response
    {
        return $this->render('maps/leaflet-maps.html.twig', [
            'controller_name' => 'MapsController',
        ]);
    }

    #[Route('/vector-maps', name: 'app_vector_maps')]
    public function vectorMaps(): Response
    {
        return $this->render('maps/vector-maps.html.twig', [
            'controller_name' => 'MapsController',
        ]);
    }
    
}

File: Dashboard Template/vyzor-dotnet/mvc/final/Controllers/MapsController.cs
using System.Diagnostics;
using Microsoft.AspNetCore.Mvc;
using vyzor.Models;

namespace vyzor.Controllers;

public class MapsController : Controller
{
    private readonly ILogger<MapsController> _logger;

    public MapsController(ILogger<MapsController> logger)
    {
        _logger = logger;
    }

    [Route("/google-maps")]
    public IActionResult google_maps()
    {
        return View("google_maps");
    }

    [Route("/leaflet-maps")]
    public IActionResult leaflet_maps()
    {
        return View("leaflet_maps");
    }

    [Route("/vector-maps")]
    public IActionResult vector_maps()
    {
        return View("vector_maps");
    }

}
